src/crypto/SessionKeystore.h

```diff
@@ -96,29 +116,44 @@ class SessionKeystore
      * Use HKDF as defined in the Matter specification to derive AES keys for both directions, and
      * the attestation challenge from the shared secret.
      *
-     * If the method returns no error, the application is responsible for destroying the handles
+     * If the method returns no error, the application is responsible for destroying the AES keys
      * using DestroyKey() method when the keys are no longer needed. On failure, the method must
      * release all handles that it allocated so far.
      */
     virtual CHIP_ERROR DeriveSessionKeys(const ByteSpan & secret, const ByteSpan & salt, const ByteSpan & info,
                                          Aes128KeyHandle & i2rKey, Aes128KeyHandle & r2iKey,
                                          AttestationChallenge & attestationChallenge) = 0;
+
+    /**
+     * @brief Derive session keys from the HKDF key
+     *
+     * Use HKDF as defined in the Matter specification to derive AES keys for both directions, and
+     * the attestation challenge from the HKDF key.
```

Comment:
```suggestion
     * @brief Derive session keys from a session's SharedSecret.
     *
     * Use Crypto_KDF (HKDF) primitive as defined in the Matter specification to derive symmetric 
     * session keys for both directions, and the attestation challenge from the session establishment's SharedSecret key (`kkdfKey`).
```

Perhaps rename "hkdfKey" to "sharedSecretKey".

It's just a suggestion, but the current wording is overly concrete compared to the spec's crypto primitive abstractions.

Reply:
On the other hand, we use AES instead of AEAD and other concrete names so it's hard to stay consistent with either convention, but OK, this interface is more generic than CHIPCryptoPAL primitives, so I can rename.

Reply:
Updated the comment and aligned previous ones.